src/style.css

```diff
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-style: bold;
+  font-weight: 900;
+}
+
+.form {
+  position: relative;
+  margin: 20px 0;
+}
+
+.magnifier1 {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  height: 19px;
+}
+
+.magnifier2 {
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  height: 11px;
+}
+
+.searchbar {
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  font-family: Avenir, sans-serif;
```

Comment:
It's better to add default font-family for html tag and use font-family: inherit for inputs